docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Create a Fabric workspace for FinOps
+description: This article walks you through creating and configuring a Microsoft Fabric workspace for FinOps.
+author: bandersmsft
+ms.author: banders
+ms.date: 07/17/2024
+ms.topic: how-to
+ms.service: finops
+ms.reviewer: micflan
+---
+
+# Create a Fabric workspace for FinOps
+
+This article walks you through creating and configuring a [Microsoft Fabric](/fabric/get-started/microsoft-fabric-overview) workspace for FinOps. It provides a step-by-step guide to create and configure Azure Data Lake storage (ADLS) setup, data export, workspace creation, and data ingestion. When completed, you can use Power BI to build reports.
+
+## Prerequisites
+
+Before you begin, you must have:
+
+- A paid Azure subscription with cost and usage information
+
+To complete this walkthrough, you create the following resources that incur costs:
+
+- An Azure Data Lake Storage Gen2 account
+- A Fabric or Power BI workspace
```

Comment:
Doesn't this require Fabric explicitly? I don't think PBI only works, right?

Reply:
Right. I"ve fixed.